docs/man/docker.1.md

```diff
@@ -90,7 +90,7 @@ unix://[/path/to/socket] to use.
   Set key=value labels to the daemon (displayed in `docker info`)
 
 **--log-driver**="*json-file*|*syslog*|*none*"
```

Comment:
Since you hit the line underneath, any objects to making it match what you have added to cli.md?